docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2'
```

Comment:
Moved to the root directory since now it boots both the backend and the frontend in development mode.